source/_data/sidebar.yml

```diff
@@ -122,6 +122,7 @@ api:
     siblings: siblings.html
     spread: spread.html
     spy: spy.html
+    state: state.html
```

Comment:
Halp! I _could not_ get this to render nicely in the sidebar. What am I missing? 😰

<img width="286" alt="screen shot 2018-07-29 at 18 39 00" src="https://user-images.githubusercontent.com/1855109/43369169-07efef80-9361-11e8-9849-393090559a26.png">